Dockerfile

```diff
@@ -41,6 +36,9 @@ ADD config /app/config
 ADD app /app/app
 ADD lib /app/lib
 
+ADD REVISION /
```

Comment:
@jacobat do we need REVISION in the root for some reason / is just app good enough ?